### PR TITLE
workspace without references

### PR DESCRIPTION
**Pull request by ericblandet**
still lacks reference to users. 
but will come

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   root to: 'pages#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :workspaces
```
Comment (Simon-Busch) [inline config/routes.rb:5]: Do we need to have all the routes generates for workspace ?

File: db/migrate/20201123130609_create_workspaces.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class CreateWorkspaces < ActiveRecord::Migration[6.0]
+  def change
+    create_table :workspaces do |t|
+      t.string :name
+      t.string :description
+      t.string :address
+      t.integer :capacity
+      t.float :daily_rate
```
Comment (Simon-Busch) [inline db/migrate/20201123130609_create_workspaces.rb:8]: shouldn't be an integer ?

File: db/schema.rb [hunk 3]
```diff
@@ -10,9 +10,19 @@
...
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "workspaces", force: :cascade do |t|
+    t.string "name"
+    t.string "description"
+    t.string "address"
+    t.integer "capacity"
+    t.float "daily_rate"
```
Comment (Simon-Busch) [inline db/schema.rb:23]: Shouldn't be an integer ?

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,8 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Workspace.destroy_all
+
+Workspace.create(name:"studio 506", description:"best office in LS", address:"avenue de Sévelin, Lausanne", capacity: 4, daily_rate:25)
+Workspace.create(name:"The Office", description:"where creativity comes true", address:"Chemin du closel, Lausanne", capacity: 12, daily_rate:50)
```
Comment (Simon-Busch) [inline db/seeds.rb:12]: Did you crash test ?

**Comment by ericblandet** [comment]
refused

**Pull request closed without merging**